### DISCUSSION TITLE
Stats API: add an `all_time` period option

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#2081] [Anything Else]
Currently, to fetch all time data from Stats API you'd need to fix a start date and generate today's date every time. Would be much simpler to just allow an `all_time` period option.

**Comment by lloydjatkinson** [comment]
Just hit this too. It's definitely needed!

**Reply by Vrq** [reply]
+1

**Reply by Vrq** [reply]
Why is it not possible to implement easily? The dashboard shows all time stats?
